Adjust CamelCase names (#160)
MikeBishop/dns-alt-svc: draft-ietf-dnsop-svcb-https.md

but those references should be taken to apply to SVCB, HTTPS,
and any future SVCB-compatible RR types.

The SVCB RR has two forms: 1) the "Alias Form" simply delegates operational
control for a resource; 2) the "Service Form" binds together
configuration information for a service endpoint.
The Service Form provides additional key=value parameters
within each RDATA set.

> 

Reviewer: While repetitive with what is below, this does provide important background context.  Should we see if we can include this even if we condense this down some?  Or just leave this as-is?

applies equally to the HTTPS RR which shares
the same encoding, format, and high-level semantics.)

The SVCB RR has two modes: AliasMode, which aliases a name to another name,
and ServiceMode, which provides connection information bound to a service
endpoint domain.  Placing both forms in a single RR type allows clients to
fetch the relevant information with a single query.

The SVCB RR has two mandatory fields and one optional.  The fields are:

1. SvcPriority: The priority of this record (relative to others,
   with lower values preferred).  A value of 0 indicates AliasMode.
   (Described in {{pri}}.)
2. TargetName: The domain name of either the alias target (for
   AliasMode) or the alternative service endpoint (for ServiceMode).

> 3. SvcParams (optional): A list of key=value pairs
>    describing the alternative service endpoint at
>    TargetName (only used in ServiceMode and otherwise ignored).

Reviewer: ```suggestion
3. SvcParams (optional): A list of key=value pairs
   describing the alternative endpoint at
   TargetName (only used in ServiceMode and otherwise ignored). 
   Described in {{svcparams}}.
```

requiring origin domains to configure and maintain an additional
delegation.

> Following of HTTPS RR aliases and CNAME aliases is unchanged from SVCB.

Reviewer: ```suggestion
Following of HTTPS AliasMode RRs and CNAME aliases is unchanged from SVCB.
```

A registration MUST include the following fields:

> * Number: Service parameter key numeric identifier (range 0-65535)

Reviewer: ```suggestion
* Number: SvcParamKey wire format numeric identifier (range 0-65535)
```

* Name: Service parameter key name
* Meaning: a short description
* Pointer to specification text

> SvcParamKey values to be added to this namespace

Reviewer: ```suggestion
SvcParamKey entries to be added to this namespace
```
